### ISSUE TITLE
File transfer: "invalid filename" error on valid filename

### ISSUE THREAD

**Issue by laniakea64** [netblue30/firejail#2307]
Xubuntu 18.04 64-bit
firejail self build from https://github.com/netblue30/firejail/commit/5e153e6793b3eb30ee58cbc8ffaf7d18d37c6dbd

```
$ firejail --version
firejail version 0.9.57

Compile time support:
        - AppArmor support is enabled
        - AppImage support is enabled
        - chroot support is enabled
        - file and directory whitelisting support is enabled
        - file transfer support is enabled
        - networking support is enabled
        - overlayfs support is enabled
        - private-home support is enabled
        - seccomp-bpf support is enabled
        - user namespace support is enabled
        - X11 sandboxing support is enabled

```

**Steps to reproduce**

1) open a web browser in a `firejail --overlay-tmpfs` sandbox, go to https://git.hyperbola.info:50100/software/icedove-uxp.git/commit/?id=7339b4e15daff465a2a5ec7448a046773f03261d

2) Where it says "commit	7339b4e15daff465a2a5ec7448a046773f03261d (patch)", right-click "patch" and save the link target in `~/Downloads`.  Don't change the filename when saving.

3) Try to transfer the file out of the sandbox with `firejail --get=<pid> ~/Downloads/de870b40934d33a433dfa97beb753ccdb6227693..7339b4e15daff465a2a5ec7448a046773f03261d.patch`

**Expected results**

File should transfer

**Actual results**

File transfer fails with this error -
```
Error: invalid file name /home/<me>/Downloads/de870b40934d33a433dfa97beb753ccdb6227693..7339b4e15daff465a2a5ec7448a046773f03261d.patch
```
Changing the `..` in the filename to `.` gets it working.

**Comment by rusty-snake** [comment]
still an issue?

**Comment by laniakea64** [comment]
Yes, still an issue in self build from https://github.com/netblue30/firejail/commit/5c20f4f53e647f6d1026f8b134fa364e96923aaf
